### PR TITLE
TNO-197 Update error handling

### PR DESCRIPTION
**Pull request by Fosol**
Updated error toast style.  Also added success toast to Data Source management form.

I believe I have also resolved a warning that shows up when state changes incorrectly within <UserInfo/>.

image

image

### CODE REVIEW

File: app/editor/src/features/admin/sources/DataSource.tsx [hunk 1]
```diff
@@ -36,6 +37,7 @@ export const DataSource: React.FC<IDataSourceProps> = (props) => {
       parentId: values.parentId ? values.parentId : undefined,
     });
     setSource({ ...data, parentId: data.parentId ? data.parentId : undefined });
+    toast.success(`${data.name} has successfully been updated.`);
```
Comment (Fosol) [inline app/editor/src/features/admin/sources/DataSource.tsx:40]: Success toast.

File: app/editor/src/features/login/UserInfo.tsx [hunk 2]
```diff
@@ -9,16 +9,20 @@ import { useKeycloakWrapper } from 'tno-core';
...
   const keycloak = useKeycloakWrapper();
-  const [{ userInfo }, app] = useApp();
+  const [, app] = useApp();
   const [, appStore] = useAppStore();
 
+  const [init, setInit] = React.useState(true);
+
   React.useEffect(() => {
-    if (keycloak.authenticated && userInfo === undefined) {
+    if (keycloak.authenticated && init) {
```
Comment (Fosol) [inline app/editor/src/features/login/UserInfo.tsx:18]: I think this resolves the weird state bug.

File: app/editor/src/store/hooks/app/useToastError.ts [hunk 3]
```diff
@@ -19,7 +19,7 @@ export const useToastError = () => {
 
   React.useEffect(() => {
     errors.forEach((e) => {
-      toast(e.message, {});
+      toast.error(e.message, {});
```
Comment (Fosol) [inline app/editor/src/store/hooks/app/useToastError.ts:22]: Change to error style.

**Review by jdtoombs — APPROVED** [review]
I like!

**Pull request merged**